integration_test/third_party_apps_data/agent/ops-agent/linux/enable_wildfly

```diff
@@ -0,0 +1,31 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logging:
+  receivers:
+    wildfly_server:
+      type: wildfly_server
+  service:
+    pipelines:
+      wildfly:
+        receivers:
+          - wildfly_server
+EOF
+
+sudo service google-cloud-ops-agent restart
```

Comment:
Do we need to add in a sleep here? Looks like we do it for some of the other applications